samples/house_price_estimates/submit_result/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.7
```

Comment:
we have 4 Dockerfiles, with a difference in the name of the running script only, can they be combined to a single Dockerfile ?

Reply:
They could but these are meant to be standalone components that could in theory be provided by different developers/organizations.  This is the standard way of packaging a kfp component.